.github/workflows/pre_merge.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       - name: Install dependencies
         run: python -m pip install -r requirements/dev.txt
       - name: Run unit test
-        run: tox -e pre-merge-all-${{ matrix.tox-env }} -- tests/unit
+        run: tox -vv -e pre-merge-all-${{ matrix.tox-env }} -- tests/unit
```

Comment:
What does the `-vv` mean?

Reply:
I've added this to increase verbosity to the tox logging. As i've shared before there are tox packaging processes are remained in the CI instances so that I want to see the detailed log messages from the tox packaging phase.